example/test_solrcloud.yaml

```diff
@@ -3,6 +3,13 @@ kind: SolrCloud
 metadata:
   name: example
 spec:
+  dataPvcSpec:
```

Comment:
Need to have persistent PVCs for test_solrcloud so that when docker locally restarts, collections and replica data persists nicely, otherwise collections and shards will remain down.